### PR TITLE
[Edit] HTML Add Media: Links

### PR DESCRIPTION
**Pull request by htnweb**
### Description

- Added images to links.md
- Added subsequent media to content/media

### Issue Solved

- Closes Issue #3231 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by cigar-galaxy82** [comment]
Hey @htnweb 

Thank you for contributing to Codecademy Docs!

I've suggested a few changes modify them at your convenience

Thank you!

**Comment by htnweb** [comment]
Hi @cigar-galaxy82  thank you for the suggestions,

I am away on holiday until next Friday, I can hopefully complete when I am back.

From looking at your suggestions I should have the media link/path on a separate line, and also in a different location ?

Thanks for your help.

**Comment by shukriarahimitech** [comment]
@htnweb, I am interested in this issue.

**Comment by shukriarahimitech** [comment]
@htnweb I am interested in this issue.

**Comment by htnweb** [comment]
Hi @cigar-galaxy82 I have uploaded the images and changed the file paths in links.md

Thanks

**Comment by letsbuildanything** [comment]
@cigar-galaxy82 Please approve this.

**Review by cigar-galaxy82 — APPROVED** [review]

File: content/html/concepts/links/links.md [hunk 1]
```diff
@@ -14,13 +14,14 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-**Links** are used in HTML to add features to a web page including styling, functionality, hyperlinks, and navigation.
+**Links** are used in HTML to add features to a web page including styling, functionality, hyperlinks, and navigation. ![Links image](https://raw.githubusercontent.com/Codecademy/docs/main/media/links.png)
 
-## Hyperlinks
+
+## Hyperlinks ![Hyperlink image](https://raw.githubusercontent.com/Codecademy/docs/main/media/hyperlink.png)
```
Comment (letsbuildanything) [inline content/html/concepts/links/links.md:20]:
```suggestion
## Hyperlinks
```

Comment (letsbuildanything) [reply content/html/concepts/links/links.md:20]: @htnweb Please remove all the image links added next to headings like this.

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@htnweb 
I've listed below some changes. Please take a moment to address them. If you have any concerns or reasons to reject specific suggestions, feel free to discuss them.
1. Ensure that the images are screenshots of the outputs generated by the given HTML code in the browser, and place them below or after the given HTML code.

2. please upload all the required images to the [/media folder](https://github.com/Codecademy/docs/tree/main/media).

3. Please remove all the unnecessary images and replace them with appropriate ones.

**Comment by ishg-153** [comment]
@htnweb It's been a week since @letsbuildanything suggested changes. Are you still working on them?

**Comment by htnweb** [comment]
Hi , I've been a bit busy on another project. I will try and complete this on Friday.

Kind regards

**Comment by htnweb** [comment]
@letsbuildanything  Hi, I think I have done the changes you suggested although I am unsure on if the file path's are correct.

Thanks

**Comment by ishg-153** [comment]
@letsbuildanything please look through this PR and approve changes.

**Comment by letsbuildanything** [comment]
Hi @htnweb,
Please use this entry: https://github.com/Codecademy/docs/blob/main/content/html/concepts/lists/lists.md as a reference on what and how to include media or screenshots in this entry. 
- The output image should be the screenshot of the HTML code output. 
-  Images should not be added in-line with the paragraph.
- No images seem to be appropriate here. So please remove all of them.

If you have any questions or need further clarification, feel free to ask.

**Comment by htnweb** [comment]
Hi @letsbuildanything I have added a screenshot of the html output, and removed the wrong media.

Thanks

**Review by letsbuildanything — APPROVED** [review]
LGTM! 🚀

**Comment by htnweb** [comment]
Thanks for everyone's suggestions on this issue, was good fun. I will now delete the branch.

**Pull request merged**